### ISSUE TITLE
Domain name expired

### ISSUE THREAD

**Issue by misbach** [codeforutah/openslc.org#1]
The website isn't working. Looks like the domain name expired.

**Comment by todrobbins** [comment]
@misbach that's correct. The new URL is https://opensaltlake.org/ and runs on WordPress.org. This is an old repo for the original website.
